packages/core/src/blocks/HeadingBlockContent/HeadingBlockContent.ts

```diff
@@ -130,7 +130,9 @@ const HeadingBlockContent = createStronglyTypedTiptapNode({
           }
 
           return {
-            level: element.getAttribute("data-level"),
+            level:
```

Comment:
do we have unit tests for other parse scenarios? if so, add the cases that we missed there?